### ISSUE TITLE
IDEA GATHERING: Back button causes navigation issues when you've opened idea(s)

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#1658]
**What is the problem?**
In this video recording, I had just opened and closed an idea, and was trying to go back to the group in which the topic sits, using the browser back button (as the 'back' option in the platform takes you to My topics, or Public groups once [this issue](https://github.com/citizenos/citizenos-fe/issues/1525) is fixed, neither of which was where I wanted to go.

Upon clicking the back button:
- First click, it takes me back to the idea I had open (which I wasn't expecting)
- Second click, nothing happens
- Third click, it takes me back to the group (where I wanted to be), but with the idea still open
- I then close the idea, but doing so takes me back to the topic again.  
- Forth click, I get to where I wanted to be (in the group, no idea pop-up)

https://www.loom.com/share/8a18c07d6b1c4ed5811d12e229418059?sid=dd986c40-d1f4-43a5-8362-df8f16a48067

**Why is this a problem?**
Disorienting and annoying for users. 

**Possible solution.**
Obviously there's some bugginess, but @kevincrepin, Lydia and @TiinaDUX, am I right in thinking that standard UX is that the back button on browser shouldn't take you through any pop-ups you've had open? I.e. in this case, rather than opening the idea again, the back button should have just taken me to the group, rather than back through the idea I had opened (and potentially through who knows how many other ideas a user might have opened and closed since arriving into the topic).

**Comment by kevincrepin** [comment]
Ye, this is a bit unexpected behaviour. My thought is that this has to do with the perception that a popup is a temporary state  and even if you would go through several of the ideas like this, you would still expect that the "root page" is the same page and one click on the browser back button would take you to the previous "root page" that you came from.

But @ilmartyrk needs to pitch in here and say whether it's possible to open up these ideas in such a way that the back/fwd browser button ignores this?

**Comment by BeccaMelhuish** [comment]
@ilmartyrk is this a simple fix? And please provide a rough estimate :)

**Comment by BeccaMelhuish** [comment]
@ilmartyrk tested in dev and I still get the published idea pop-up opening and then staying open when I use the browser back button (in Chrome), so putting it back to in progress :)

**Comment by ilmartyrk** [comment]
@BeccaMelhuish yeah, it was not merged yet, was waiting on review, but will merge, as it is actually a really few lines of code

**Comment by BeccaMelhuish** [comment]
I tested and this is definitely better than it was. The idea does still open but it doesn't get 'stuck' open anymore. So let's put the code through to pre-live, though the ideal would be that the back button doesn't open up the idea at all. Not sure if that's possible though @ilmartyrk? If it is I will make a separate issue to do that too :)

**Comment by ilmartyrk** [comment]
@BeccaMelhuish if the previous page is idea view, it should open the idea pop-up. History management is quite difficult to solve so I would not go into that

**Comment by BeccaMelhuish** [comment]
OK noted, thank you, let's leave it how it is then! :)
